Psychtoolbox/PsychOpenGL/PsychGLSLShaders/SmoothedDiscShader.frag.txt

```diff
@@ -28,16 +29,21 @@ void main()
     Dist = distance(pos, Center);
 
     /* If distance to center (aka radius of pixel) > Radius, discard this pixel: */
-    if (Dist > Radius) discard;
+    /*if (Dist > Radius) discard;*/
```

Comment:
?

Reply:
The reason is if we use `(Dist > Radius) discard` then the inverse smoothstep method (to create a mask) fails. So I've now added a `if (Method < 2.0)` to make sure this is always applied except for when we want an inverse smoothstep.